fastlane/README.md

```diff
@@ -0,0 +1,23 @@
+fastlane documentation
```

Comment:
I see that this is autogenerated. Documenting the ENV variables in a readme would be helpful.